.travis.yml

```diff
@@ -2,6 +2,8 @@ language: go
 
 matrix:
   include:
+  - go: 1.13.x
+    env: GO111MODULE=on GOSUBMODULE=on
```

Comment:
Would you mind adding the tests of this module to the "TESTEXTRAS" target instead?  Every target here starts up a new VM so I'd like to keep the list limited.

Reply:
Done.

Reply:
Done.